Expensify/App
# Resync data when the app reconnects

Fixes https://github.com/Expensify/ReactNativeChat/issues/253

# Tests
1. Load the app
1. Go offline (either turn off wifi or set the browser to offline mode)
1. Post a comment
1. In another browser, go to expensify.com and go to the same report that is open for the chat just commented on
1. Post a comment on the report page
1. Now switch back to the chat app and turn it back to online mode
1. Verify that you see the comment from `5` show up
